kvmagent/kvmagent/test/localstorage_testsuite/test_data_volume_1.py

```
@@ -0,0 +1,117 @@
+
+            return data_vol
+
+        # data_vol = copy.deepcopy(start_vm_data_vol)
+        # data_vol["volumeUuid"] = vol_uuid
+        # data_vol["resourceUuid"] = vol_uuid
+        # data_vol["installPath"] = vol_path
+        # body = copy.deepcopy(startVmCmdBody)
+        #
+        # body["dataVolumes"] = vols_body
+
+        deviceAddress = {
```

Ruff (0.12.2)

Local variable `deviceAddress` is assigned to but never used

Remove assignment to unused variable `deviceAddress`

(F841)